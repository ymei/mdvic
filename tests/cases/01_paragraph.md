This is a simple paragraph that should wrap around the configured width without any styling.

