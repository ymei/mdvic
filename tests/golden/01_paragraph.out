This is a simple paragraph that should w
rap around the configured width without 
any styling.

